lib/ansible/executor/play_iterator.py

```diff
@@ -181,6 +185,8 @@ def __init__(self, inventory, play, play_context, variable_manager, all_vars, st
         }
         if gather_timeout:
             setup_task.args['gather_timeout'] = gather_timeout
+        if fact_path:
+            setup_task.args['fact_path'] = fact_path
```

Comment:
Can I go ahead and move the defaulting to play_context or should I wait on an opinion on Play vs PlayContext by @jimi-c?